### PR TITLE
Support force_tuple argument

### LINKED ISSUE #932: SplitAxis should return a tuple with a single element
`Function.__call__` returns a `Variable` when its forward function returns only one return value. When the number of return values depends on input values, such as `split_axis`, a user need to check the type of the return value of `Function`. A user needs to write a complicated source code.

I have two design choice:
- Only `SplitAxis` always returns a tuple of `Variable`s
- Add keyword argument `force_tuple` to `Function.__call__`

### PR DESCRIPTION
**Pull request by unnonouno**
Fix #932

### CODE REVIEW

File: chainer/functions/array/split_axis.py [hunk 1]
```diff
@@ -68,11 +69,14 @@ def split_axis(x, indices_or_sections, axis):
...
             indicates the positions where the array is split.
         axis (int): Axis that the input array is split along.
+        force_tuple (bool): If True, this method returns a tuple even when the
+            number of outputs is one.
 
     Returns:
         ``tuple`` or ``Variable``: Tuple of :class:`~chainer.Variable` objects
              if the number of outputs is more than 1 or
              :class:`~chainer.Variable` otherwise.
+             When `force_tuple` is True, return value is always a tuple.
```
Comment (delta2323) [inline chainer/functions/array/split_axis.py]: the returned value

Comment (delta2323) [inline chainer/functions/array/split_axis.py]: It gets easier to understand if we add `regardless of the number of outputs` at the end of the sentence.

**Comment by delta2323** [comment]
I added comments. Please check them.

**Comment by unnonouno** [comment]
I fixed the dostring.

**Comment by delta2323** [comment]
LGTM :+1:

File: chainer/functions/array/split_axis.py [hunk 2]
```diff
@@ -68,11 +69,15 @@ def split_axis(x, indices_or_sections, axis):
             If it is a 1-D array of sorted integers, it
             indicates the positions where the array is split.
         axis (int): Axis that the input array is split along.
+        force_tuple (bool): If True, this method returns a tuple even when the
```
Comment (delta2323) [inline chainer/functions/array/split_axis.py]: "``True``"

**Pull request merged**